syllabus-app/src/webapp/syllabus/css/jqueryui-editable.css

```diff
@@ -136,7 +136,7 @@
 }
 
 .editable-pre-wrapped {
-   white-space: pre-wrap;
```

Comment:
Much better off to override this CSS iin syllabus.css as the jquery-editable.css is taken as-is from an upstream source and if a new version of the library gets pulled in this change will probably get lost.